packages/datatrak-web/src/api/api.ts

```diff
@@ -6,7 +6,8 @@
 import axios from 'axios';
 import FetchError from './fetchError';
 
-export const API_URL = import.meta.env.REACT_APP_DATATRAK_WEB_API_URL || 'http://localhost:8110/v1';
+// Needs to use process.env instead of import.meta.env for compatibility with jest
+export const API_URL = process.env.REACT_APP_DATATRAK_WEB_API_URL || 'http://localhost:8110/v1';
```

Comment:
Will this still work when running the app ?

Reply:
Yes I double checked. Also this is what we do in psss which is re-assuring. I think it was you who added capability for this, right? By adding `loadEnv` in the vite config.